app/lib/meadow/data/indexer.ex

```diff
@@ -40,6 +40,18 @@ defmodule Meadow.Data.Indexer do
     end)
   end
 
```

Comment:
Is it maybe worth rewriting `reindex_all/1` to call your new `reindex_all/2` instead of having the code repeated?

```suggestion
  def reindex_all(version) do
    reindex_all(2, [FileSet, Work, Collection])
  end
```

You'd need to leave the special case `reindex_all(1)` alone since the hot swap is outside the loop, but that's going away soon anyway.

Reply:
Thanks for calling this out, hopefully that does the trick.